lib/cereal/builders/entity.ex

```diff
@@ -100,13 +100,9 @@ defmodule Cereal.Builders.Entity do
   end
   defp filter_attributes(attrs, _), do: attrs
 
-  defp do_filter_attributes(attrs, nil), do: attrs
   defp do_filter_attributes(attrs, {:take, fields}) when is_list(fields),
     do: Map.take(attrs, fields)
   defp do_filter_attributes(attrs, {:drop, fields}) when is_list(fields),
     do: Map.drop(attrs, fields)
-  defp do_filter_attributes(attrs, {action, fields}) when is_binary(fields) do
-    fields = fields |> String.split(",") |> Enum.map(&String.to_atom/1)
```

Comment:
do we not still have to do this? with `String .to_existing_atom/1`?

Reply:
No we just do so elsewhere. Rather than parse the `include` option into atoms at the beginning of serialization, but parse the `fields` and `excludes` options into atoms here, we do so at the beginning for all.

See

https://github.com/Frameio/cereal_ex/blob/bc83d964ee7f5b93e42f046b9777aefa90950597/lib/cereal/builders/base.ex#L48-L53

And

https://github.com/Frameio/cereal_ex/blob/bc83d964ee7f5b93e42f046b9777aefa90950597/lib/cereal/utils.ex#L51-L74